packages/frontend/src/components/Settings/Notifications.tsx

```diff
@@ -17,12 +17,32 @@ type Props = {
 
 export default function Notifications({ desktopSettings }: Props) {
   const tx = useTranslationFunction()
+  const accountId = selectedAccountId()
+  const { accounts } = useAccountNotificationStore()[0]!
+  const isMuted = accounts[accountId]?.muted || false
+
+  function AccountNotificationMutedSwitch({
+    label,
+    description,
+  }: {
+    label: string
+    description?: string
+  }) {
+    return (
+      <SettingsSwitch
+        label={label}
+        value={isMuted}
+        description={description}
+        disabled={!desktopSettings['notifications']}
+        onChange={() => {
+          AccountNotificationStoreInstance.effect.setMuted(accountId, !isMuted)
+        }}
+      />
+    )
+  }
```

Comment:
Apparently defining a component like this inline makes React create a new DOM element on every re-render, which is apparent when you toggle the value of this switch with keyboard: it will lose focus.

Reply:
Fixed it. There is anyway no need to render it inline